.github/workflows/lint.yml

```diff
@@ -9,17 +9,12 @@ on:
       - '.github/workflows/lint.yml'
 jobs:
   xo:
-
-    runs-on: [ubuntu-latest]
-
-    strategy:
-      matrix:
-        node-version: [12.x]
-
+    runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js
       uses: actions/setup-node@v1
       with:
-        node-version: ${{ matrix.node-version }}
+        node-version: 14
```

Comment:
might wanna do `latest` here.

Reply:
I believe GitHub Action doesn't have `latest` and `lts`